lambdas/service/app.py

```diff
@@ -2283,15 +2335,46 @@ def get_data_object(file_uuid):
         return Response(response.text, status_code=response.status_code)
 
 
-@app.route(drs.drs_http_object_path('{file_uuid}', access_id='{access_id}'), methods=['GET'], cors=True)
-def get_data_object_access(file_uuid, access_id):
-    """
-    Return a DRS data object dictionary for a given DSS file UUID, version, and
-    access ID.
+@app.route(drs.drs_http_object_path('{file_uuid}', access_id='{access_id}'), methods=['GET'], cors=True, method_spec={
+    'summary': 'Get a file with an access ID',
+    'description': format_description('''
+        This endpoint returns a URL that can be used to fetch the bytes of a DRS
+        object.
 
-    The access ID endpoint is only necessary if the DSS needs to do a checkout
-    for the file.
-    """
+        This method only needs to be called when using an `AccessMethod` that
+        contains an `access_id`.
+
+        An `access_id` is returned when the underlying file is not ready. When
+        the underlying repository is the DSS, the 202 response allowed time for
+        the DSS to do a checkout.
+    ''') + drs_description,
+    'parameters': [
+        params.path('file_uuid', str, description='UUID of the file to be fetched'),
+        params.path('access_id', str, description='Access ID returned from a previous request'),
+    ],
+    'responses': {
+        '202': {
+            'description': format_description('''
+                This response is issued if the object is not yet ready. Respect
+                the `Retry-After` header, then try again.
```

Comment:
I thought there is no such header. If this is our own deviation from the standard, we should explicit declare that fact.

Reply:
This is as per the DRS spec https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/docs/#_responses